appveyor.yml

```diff
@@ -0,0 +1,40 @@
+version: 1.0.{build}
+image: Visual Studio 2017
+
+environment:
+  CCACHE_DIR: "%APPVEYOR_BUILD_FOLDER%\\.ccache"
+  CARGO_INCREMENTAL: 0
+  RUST_BACKTRACE: 1
+  RUSTFLAGS: -Dwarnings
+
+  matrix:
+  - TARGET: nightly-x86_64-pc-windows-msvc
+
+branches:
+  only:
+    - master
+
+cache:
+  - .ccache
+
+install:
+  - appveyor-retry choco install pkgconfiglite
+  - appveyor-retry appveyor DownloadFile https://gstreamer.freedesktop.org/data/pkg/windows/1.16.0/gstreamer-1.0-devel-msvc-x86_64-1.16.0.msi
+  - appveyor-retry appveyor DownloadFile https://gstreamer.freedesktop.org/data/pkg/windows/1.16.0/gstreamer-1.0-msvc-x86_64-1.16.0.msi
+  - msiexec /i gstreamer-1.0-devel-msvc-x86_64-1.16.0.msi /quiet /qn /norestart /log install-devel.log ADDLOCAL=ALL
+  - msiexec /i gstreamer-1.0-msvc-x86_64-1.16.0.msi /quiet /qn /norestart /log install.log ADDLOCAL=ALL
+  - call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars64.bat"
+  - appveyor-retry appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
+  - rustup-init.exe -y --default-host x86_64-pc-windows-msvc --default-toolchain nightly
+  - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
+  - set PKG_CONFIG_PATH=%PKG_CONFIG_PATH%;C:\gstreamer\1.0\x86_64\lib\pkgconfig
+  - set LIB=%LIB%;C:\gstreamer\1.0\x86_64\lib
+  - rustup -V
+  - rustc --version
+  - cargo --version
+
+
+build_script:
+  - cargo build --all-features
```

Comment:
shouldn't we run the examples? or is spoofing audio output hard?

Reply:
Yes, that would be convenient, I'll see if it's possible to run them as a follow-up.